### ISSUE TITLE
CI: /t/02-test_ocr.t fails on Tumbleweed due tesseract-ocr update to 5.3.0

### ISSUE THREAD

**Issue by pevik** [os-autoinst/os-autoinst#2248]
openSUSE Leap 15.4 has versions which work:
https://build.opensuse.org/build/devel:openQA:GitHub:os-autoinst:os-autoinst:PR-2240/openSUSE_Leap_15.4/x86_64/os-autoinst/_log
```
tesseract-ocr-4.1.1-bp154.1.77
tesseract-ocr-traineddata-english-3.04
...
[  220s] 3: [13:06:58] ./t/02-test_ocr.t .......................... 
[  220s] 3: ok 1 - log output for needle init
[  221s] 3: ok 2 - log output for OCR
[  221s] 3: ok 3 - ocr match 1
[  222s] 3: ok 4 - log output for tesseract call
[  222s] 3: ok 5 - log output for tesseract call
[  222s] 3: ok 6 - OCR area found
[  222s] 3: ok 7 - multiple OCR regions
[  222s] 3: ok 8 - no (unexpected) warnings (via done_testing)
[  222s] 3: 1..8
[  222s] 3: ok     2826 ms ( 0.00 usr  0.00 sys +  2.42 cusr  0.45 csys =  2.87 CPU)
[  223s] 3: [13:07:01] ./t/03-testapi.t ........................... 
```

Tumbleweed has versions which fail:
https://build.opensuse.org/build/devel:openQA:GitHub:os-autoinst:os-autoinst:PR-2240/openSUSE_Tumbleweed/x86_64/os-autoinst/_log
```
tesseract-ocr-5.3.0-1.1
tesseract-ocr-traineddata-english-3.04.00-4.17
...
[  271s] 3: [17:21:31] ./t/02-test_ocr.t .......................... 
[  271s] 3: ok 1 - log output for needle init
[  271s] 3: not ok 2 - log output for OCR
[  271s] 3: ok 3 - ocr match 1
[  271s] 3: 
[  271s] 3: #   Failed test 'log output for OCR'
[  271s] 3: #   at ./t/02-test_ocr.t line 36.
[  271s] 3: # STDERR:
[  271s] 3: # Error opening data file /usr/share/tesseract-ocr/tessdata/eng.traineddata
[  271s] 3: # Please make sure the TESSDATA_PREFIX environment variable is set to your "tessdata" directory.
[  271s] 3: # Failed loading language 'eng'
[  271s] 3: # Tesseract couldn't load any languages!
[  271s] 3: # Could not initialize tesseract.
[  271s] 3: # readline() on closed filehandle $fh at /home/abuild/rpmbuild/BUILD/os-autoinst-4.6.1674046815.3a55443/ocr.pm line 23.
[  271s] 3: # Error opening data file /usr/share/tesseract-ocr/tessdata/eng.traineddata
[  271s] 3: # Please make sure the TESSDATA_PREFIX environment variable is set to your "tessdata" directory.
[  271s] 3: # Failed loading language 'eng'
[  271s] 3: # Tesseract couldn't load any languages!
[  271s] 3: # Could not initialize tesseract.
[  271s] 3: # readline() on closed filehandle $fh at /home/abuild/rpmbuild/BUILD/os-autoinst-4.6.1674046815.3a55443/ocr.pm line 23.
[  271s] 3: # 
[  271s] 3: # doesn't match:
[  271s] 3: # (?^u:Tesseract.*OCR)
[  271s] 3: # as expected
[  272s] 3: not ok 4 - log output for tesseract call
[  272s] 3: 
[  272s] 3: #   Failed test 'log output for tesseract call'
[  272s] 3: #   at ./t/02-test_ocr.t line 42.
[  272s] 3: # STDERR:
[  272s] 3: # Error opening data file /usr/share/tesseract-ocr/tessdata/eng.traineddata
[  272s] 3: # Please make sure the TESSDATA_PREFIX environment variable is set to your "tessdata" directory.
[  272s] 3: # Failed loading language 'eng'
[  272s] 3: # Tesseract couldn't load any languages!
[  272s] 3: # Could not initialize tesseract.
[  272s] 3: # readline() on closed filehandle $fh at /home/abuild/rpmbuild/BUILD/os-autoinst-4.6.1674046815.3a55443/ocr.pm line 23.
[  272s] 3: # Use of uninitialized value in concatenation (.) or string at ./t/02-test_ocr.t line 42.
[  272s] 3: # 
[  272s] 3: # doesn't match:
[  272s] 3: # (?^u:Tesseract.*OCR)
[  272s] 3: # as expected
[  272s] 3: not ok 5 - log output for tesseract call
[  272s] 3: 
[  272s] 3: #   Failed test 'log output for tesseract call'
[  272s] 3: #   at ./t/02-test_ocr.t line 42.
[  272s] 3: # STDERR:
[  272s] 3: # Error opening data file /usr/share/tesseract-ocr/tessdata/eng.traineddata
[  272s] 3: # Please make sure the TESSDATA_PREFIX environment variable is set to your "tessdata" directory.
[  272s] 3: # Failed loading language 'eng'
[  272s] 3: # Tesseract couldn't load any languages!
[  272s] 3: # Could not initialize tesseract.
[  272s] 3: # readline() on closed filehandle $fh at /home/abuild/rpmbuild/BUILD/os-autoinst-4.6.1674046815.3a55443/ocr.pm line 23.
[  272s] 3: # Use of uninitialized value in concatenation (.) or string at ./t/02-test_ocr.t line 42.
[  272s] 3: # 
[  272s] 3: # doesn't match:
[  272s] 3: # (?^u:Tesseract.*OCR)
[  272s] 3: # as expected
[  272s] 3: ok 6 - OCR area found
[  272s] 3: not ok 7 - multiple OCR regions
[  272s] 3: not ok 8 - no (unexpected) warnings (via done_testing)
[  272s] 3: 
[  272s] 3: #   Failed test 'multiple OCR regions'
[  272s] 3: #   at ./t/02-test_ocr.t line 45.
[  272s] 3: 
[  272s] 3: #   Failed test 'no (unexpected) warnings (via done_testing)'
[  272s] 3: #   at ./t/02-test_ocr.t line 48.
[  272s] 3: # Got the following unexpected warnings:
[  272s] 3: #   1: readline() on closed filehandle $fh at /home/abuild/rpmbuild/BUILD/os-autoinst-4.6.1674046815.3a55443/ocr.pm line 23.
[  272s] 3: #   2: readline() on closed filehandle $fh at /home/abuild/rpmbuild/BUILD/os-autoinst-4.6.1674046815.3a55443/ocr.pm line 23.
[  272s] 3: 1..8
[  272s] 3: #   3: readline() on closed filehandle $fh at /home/abuild/rpmbuild/BUILD/os-autoinst-4.6.1674046815.3a55443/ocr.pm line 23.
[  272s] 3: #   4: Use of uninitialized value in concatenation (.) or string at ./t/02-test_ocr.t line 42.
[  272s] 3: #   5: readline() on closed filehandle $fh at /home/abuild/rpmbuild/BUILD/os-autoinst-4.6.1674046815.3a55443/ocr.pm line 23.
[  272s] 3: #   6: Use of uninitialized value in concatenation (.) or string at ./t/02-test_ocr.t line 42.
[  272s] 3: # Looks like you failed 5 tests of 8.
[  272s] 3: Dubious, test returned 5 (wstat 1280, 0x500)
[  272s] 3: Failed 5/8 subtests 
```

NOTE even pointing variable to the right directory (`export TESSDATA_PREFIX=/usr/share/tessdata`) does not help.

**Comment by perlpunk** [comment]
There is already a tiket: https://progress.opensuse.org/issues/123193
An updated version of tesseract-ocr-traineddata is on its way to Tumblweed :)

**Comment by mimi1vx** [comment]
fixed by #2249
